pkg/awsManager/ec2Manager.go

```diff
@@ -29,18 +29,14 @@ func ListEc2InstancesForDeletion(client clientpkg.Client) []*string {
 				for _, tag := range instance.Tags {
 
 					if strings.HasPrefix(*tag.Key, "kubernetes.io") && (*instance.State.Code != 48) {
-
 						EC2InstancesToBeDeleted = append(EC2InstancesToBeDeleted, instance.InstanceId)
 						break
 					}
 
 					if (*tag.Key == "clusterAccountName" || *tag.Key == "clusterClaimLink" || *tag.Key == "clusterNamespace" || *tag.Key == "clusterClaimLinkNamespace") && (*instance.State.Code != 48) {
 						EC2InstancesToBeDeleted = append(EC2InstancesToBeDeleted, instance.InstanceId)
-
 						break
 					}
-					// no need to delete the instances if the above two 'if' conditions are not true
```

Comment:
Any reason to remove the comment?

Reply:
I don't think writing comments for previous lines should be acceptable, I will add a comment above the statements explaining what is happening. Also this method needs to be unexported because it is just a helper to CleanEc2Instances()

Reply:
Please add a comment in its place then. It's explaining the code, I'm not sure the position really matters if its before or after. Regardless, I think this explanation is good to have somewhere